Packs/GCP-IAM/Integrations/GCPIAM/GCPIAM.py

```diff
@@ -1239,13 +1239,21 @@ def generate_iam_policy_command_output(response: dict, resource_name: str = None
         readable_header = f'{resource_name} IAM policy information:'
     outputs = copy.deepcopy(response)
     outputs['name'] = resource_name
+    bindings = outputs.get("bindings", [])
+    if roles and bindings:
+        bindings_roles_only = []
+        for index, entry in enumerate(bindings):
+            if entry["role"] in roles:
+                bindings_roles_only.append(bindings.pop(index))
+
+        bindings = bindings_roles_only
 
     if limit and page:
         start = (page - 1) * limit
         end = start + limit
-
-        bindings = outputs.get("bindings", [])
         outputs["bindings"] = bindings[start:end]
+        if len(bindings) < 50:
```

Comment:
> The limit default value is 50 but could be larger, why do we need to change the header?
I have updated it to use `limit`.  The default header would come from `get_pagination_readable_message` which would use `limit` and that would be an incorrect output when filtering by a list of roles. If this was not changed whe using `!gcp-iam-project-iam-policy-get project_name="projects/target_project" roles="roles/editor,roles/owner"` the example would look like:

```
Current page size: 50
Showing page 1 out of others that may exist.
```

Since there are only 2 roles ( `roles/editor, roles/owner`), it should be:

````
Current size: 2
```

> Additionally, you check the len of the original list without paging (bindings[start:end]), is that your intention?

Yes, because I would like to return all possible roles, but I could page the roles if we think a user would input over 50 roles into a list.